src/core/tests/TransformFromMatrix4x4UnitTest.cpp

```diff
@@ -36,8 +37,26 @@ void checkFromMatrix4x4()
     ASSERT_EQUAL_TRANSFORM(t_posrot, t_buffer);
 }
 
+void checkLogExp()
+{
+    PositionRaw pos_raw(0, 0, 0);
+    Rotation rot = Rotation::RPY(0.0, 1.57, 0.0);
+    Position pos = Position(pos_raw);
+    Transform t_posrot(rot, pos);
+
+    SpatialMotionVector v;
+    v.zero();
+    v.setAngularVec3(rot.log());
+
+    Transform t_v = v.exp();
+
+    ASSERT_EQUAL_TRANSFORM_TOL(t_posrot, t_v, 0.01);
```

Comment:
This test is not testing the linear part?

Reply:
Computationally, it is. Quantitatively, no. Since the linear part is set to all zeros.

Reply:
I see, we are not checking if the function is correct, but at least we are checking stuff like that memory is correct with Valgrind, etc etc.